docs/Classes/AddressView.html

```diff
@@ -42,7 +42,7 @@
     </header>
 
     <p class="breadcrumbs">
-      <a class="breadcrumb" href="../index.html">FramesIos Reference</a>
```

Comment:
Can you share the jazzy command which worked for you ? just for future reference.

Reply:
Everything is configured in `.jazzy.yaml`, so you just need to run `jazzy` 😄